plugins/web/CMakeLists.txt

```diff
@@ -26,11 +26,16 @@ VASTRegisterPlugin(
 
 find_package(OpenSSL REQUIRED)
 
-# Setup dependency code from submodules.
-list(PREPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
-add_subdirectory(aux)
-target_link_libraries(web PRIVATE restinio)
-list(POP_FRONT CMAKE_MODULE_PATH)
+if (VAST_ENABLE_SYSTEM_RESTINIO)
```

Comment:
Should this use the same logic as other optionally bundled packages, e.g., similar to `VAST_ENABLE_BUNDLED_CAF` or `VAST_ENABLE_BUNDLED_ROBIN_MAP`?

The option here is essentially undocumented, and the logic differs from what we do in `libvast/CMakeLists.txt` for the others. The naming also differs, which I find rather confusing.

Reply:
Makes sense to align with the rest.
I would put the option into the plugin CMakelists.txt and enable it by default.

Reply:
Yeah, I'm pushing a fix to this directly, we just talked about this quickly in Slack.

Reply:
@tobim mind taking a look at that last commit I just pushed and resolving this if you're fine with the change?